app/cdap/components/WranglerGrid/SelectColumnPanel/SelectedColumnCountWidget/index.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import T from 'i18n-react';
+import { SELECT_COLUMN_LIST_PREFIX } from 'components/WranglerGrid/SelectColumnPanel/constants';
+import { NormalFont } from 'components/common/TypographyText';
+
+export default function({ selectedColumnsCount }: { selectedColumnsCount: number }) {
+  const text = selectedColumnsCount
+    ? selectedColumnsCount > 10
```

Comment:
The `i18n-react` library has handling for plurals. Please see https://www.npmjs.com/package/i18n-react

Reply:
Fixed the issue.
Added T.translate(`${SELECT_COLUMN_LIST_PREFIX}.columnsSelected`, {selectedColumnsCount})
YAML file - columnsSelected: "{selectedColumnsCount} column selected"

Reply:
Please look at the documentation again. You can add strings in `text-en.yaml` that are indexed by count:
```
columnsSelected:
  0: No column selected
  1: 1 column selected
  _: {context} columns selected
```
then you can access the string with no comparisons in JS:
```
T.translate(`${SELECT_COLUMN_LIST_PREFIX}.columnsSelected`, {context: selectedColumnsCount}) 
```